src/Uri.php

```diff
@@ -91,7 +91,7 @@ public function normalizeLinks(array $links)
             } catch (\Exception $ex) {
                 continue;
             }
-            if ('' === (string) $checkUri->getHost()) {
+            if ('' === (string) $checkUri->getHost() && '/' !== $checkUri->getPath()) {
```

Comment:
filter out duplicate domain (/ was added resulted in secod request) if -d domain.tld (without /)